paasta_tools/paasta_metastatus.py

```diff
@@ -165,8 +166,14 @@ def main(argv=None):
 
         if args.autoscaling_info:
             print_with_indent("Autoscaling resources:", 2)
-            autoscaling_info = get_autoscaling_info()
-            for line in format_table(autoscaling_info):
+            headers = [field.replace("_", " ").capitalize() for field in AutoscalingInfo._fields]
+            table = reduce(
+                lambda x, y: x + [(y)],
+                get_autoscaling_info_for_all_resources(),
+                [headers]
+            )
```

Comment:
I feel like this would be more readable like:
```
table = [[field.replace("_", " ").capitalize() for field in AutoscalingInfo._fields]]
table += get_autoscaling_info_for_all_resources()
```